docs/Queries/About Queries.md

```diff
@@ -22,6 +22,7 @@ In Live Preview and Reading modes, this will list *all* tasks from your vault, r
 
 This is probably not what you want.
 Therefore, Tasks allows you to set query options to filter the tasks that you want to show.
+*All the queries are case insensitive.*
```

Comment:
Tbh I didn't even know that users can do regex searches... I didn't fully explore the plugin :/
As you know the project much better than I do, if you ever have any ideas for text for the documentation, I'd love to hear from you.

If there are places where the support for the case insensitivness is missing, please let me know :)

Reply:
> Tbh I didn't even know that users can do regex searches... I didn't fully explore the plugin :/

OK, no problem at all... And no expectation that you did - I just wanted to avoid accidentally placing unrealistic expectations, if you didn't know about regex searches.

Let's leave the documentation as it is for now. After I've merged the pull request, I'll work on the documentation a little bit - it won't take long.